cmd/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"github.com/mattn/go-colorable"
+	server "github.com/muradrmagomedov/final-project/pkg/api"
+	"github.com/muradrmagomedov/final-project/pkg/repository"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	logrus.SetFormatter(&logrus.TextFormatter{ForceColors: true})
+	logrus.SetOutput(colorable.NewColorableStdout())
+
+	config, err := loadConfig()
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	repo := repository.NewRepository()
+	err = repo.InitiateDB(config.dbPath)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	server.Repo = repo
+	server.Repo.SqliteDB(config.dbPath)
+
+	srv := new(server.Server)
+
+	err = srv.Run(config.port)
+	if err != nil {
+		logrus.Fatal("Cant start server:", err)
```

Comment:
Если прям по красоте делать, то можно 
```suggestion
		logrus.log.WithFields(log.Fields{"error": err}).Fatal("Cant start server")
```

Reply:
компилятор ругается. Пишет что у logrus нет поля log
![Снимок экрана 2024-07-16 194309](https://github.com/user-attachments/assets/08fad6a4-fa10-4755-a2cd-77fd023ee35e)

Reply:
```suggestion
		logrus.WithFields(logrus.Fields{"error": err}).Fatal("Cant start server")
```